contributed_definitions/NXaberration.nxdl.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" ?>
+<?xml-stylesheet type="text/xsl" href="nxdlformat.xsl"?>
+<definition category="base" extends="NXobject" name="NXaberration" type="group" xmlns="http://definition.nexusformat.org/nxdl/3.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.w3.org/2001/XMLSchema-instance">
+    <doc>
+         A container to store details about relevant aberrations.
```

Comment:
All these fields have terse names.  Needs some words here that these all belong to electron optics.  Documentation where these terms are defined.

Reply:
added literature and docstrings for details

Reply:
I'm not able to see any additions from GitHub's viewer.  However, there are changes visible in my local clone of this repo, once I pull the changes.